renovate.json

```
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    "group:linters"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": ["at any time"],
+    "stabilityDays": 0,
+    "dependencyDashboardApproval": true
+  },
+  "includePaths": [
+    "package.json",
+    "website/package.json"
+  ],
```

```suggestion
  ],
  "stabilityDays": 3,
```
It's possible to wait n days before opening the PR. This will reduce risks with the automerge.